kafka-sync.go

```diff
@@ -285,6 +287,7 @@ func (s *Syncer) IndexTopic(kafka sarama.Client, index diff.Index) (msgCount uin
 	go doIndex()
 
 	saveBatch := func(restart bool) {
+		glog.V(4).Info("finalize")
```

Comment:
Below comment plz ;)

Reply:
do you mean use finalize logic on kafka error ?

Reply:
No just move the below the comment ;-)